src/web/src/pages/CourseScheduler.vue

```diff
@@ -93,11 +93,15 @@
         </b-row>
       </b-col>
     </b-row>
+    <!-- NOTE: Content Class Specifies The Styling For Modal In Dark Mode. -->
     <b-modal
       id="courseInfoModal"
       v-if="courseInfoModalCourse"
       v-model="showCourseInfoModal"
       :title="courseInfoModalCourse.name + ' ' + courseInfoModalCourse.title"
+      :content-class="
+        $store.state.darkMode ? 'onStyleDarkMode' : 'offStyleDarkMode'
```

Comment:
We only need to have custom css when dark mode is active, so we don't actually need `offStyleDarkMode`. You can "toggle" classes dynamically using the object syntax for class bindings.

Reply:
What do you mean by toggle classes dynamically? Would I just remove the Else Condition?

Reply:
You can do class bindings using the object syntax, so like this `:class="{ <class name>: <boolean expr>  }"`

So given `<a :class="{ active: isActive }">` will be rendered as `<a class="active">` when `isActive === true`, and `<a>` when `isActive === false`

Reply:
Thank you. That makes sense to me.
Just curious if you know where the X is currently being styled? 
I've been trying to look for it.

Reply:
Dont think its being styled, you'll probably need to add a selector to the `dark.scss` file for the X

Reply:
Yes I agree, I didn't manage to find that either.
But is the Code for the X somewhere or is that just implicitly done by the B-Modal?